testing/ef_tests/Makefile

```
@@ -13,7 +13,7 @@ BLS_TARBALL = $(patsubst %,%-$(BLS_TEST_TAG).tar.gz,$(BLS_TEST))
 BLS_OUTPUT_DIR := $(OUTPUT_DIR)/$(BLS_TEST_REPO_NAME)
 BLS_BASE_URL := https://github.com/ethereum/$(BLS_TEST_REPO_NAME)/releases/download/$(BLS_TEST_TAG)
 
-WGET := $(if $(LIGHTHOUSE_GITHUB_TOKEN),wget --header="Authorization: $(LIGHTHOUSE_GITHUB_TOKEN)",wget)
+CURL := $(if $(LIGHTHOUSE_GITHUB_TOKEN),curl -L --header="Authorization: $(LIGHTHOUSE_GITHUB_TOKEN)",curl -L)
```

CI is failing with `curl: option --header=Authorization: ***: is unknown`, I think it needs to be a space instead of `=` after `--header` 

https://github.com/sigp/lighthouse/actions/runs/5529786759/jobs/10088333213?pr=4497

```suggestion
CURL := $(if $(LIGHTHOUSE_GITHUB_TOKEN),curl -L --header "Authorization: $(LIGHTHOUSE_GITHUB_TOKEN)",curl -L)
```

---

Good catch, thanks!